docker/cygnus-ngsi/Dockerfile

```diff
@@ -284,10 +286,7 @@ RUN \
     APACHE_DOMAIN="$(curl -s 'https://www.apache.org/dyn/closer.cgi?as_json=1' | python2 -c 'import json,sys;obj=json.load(sys.stdin);print obj["preferred"]')" \
       || APACHE_DOMAIN="http://archive.apache.org/dist/" && \
     MVN_URL="${APACHE_DOMAIN}maven/maven-3/${MVN_VER}/binaries/apache-maven-${MVN_VER}-bin.tar.gz" && \
-    # FIXME #2179: Flume version has steped to 1.10.0 so 1.9.0 is no longer available in the usual place.
-    # Temporal fix is to download it from archive.apache.org. Once we move to 1.10.0, reenable the following line
-    # FLUME_URL="${APACHE_DOMAIN}flume/${FLUME_VER}/apache-flume-${FLUME_VER}-bin.tar.gz" && \
-    FLUME_URL="https://archive.apache.org/dist/flume/stable/apache-flume-1.9.0-bin.tar.gz" && \
+    #FLUME_URL="${APACHE_DOMAIN}flume/${FLUME_VER}/apache-flume-${FLUME_VER}-bin.tar.gz" && \
```

Comment:
```suggestion
```

I understand that with this in the pom.xml files

```
    <repositories>
        <repository>
            <id>mvnrepository</id>
            <url>https://repo1.maven.org/maven2</url>
        </repository>
    </repositories>
```

The old download in Dockerfile is no longer needed